zulip/integrations/bridge_with_slack/run-slack-bridge

```diff
@@ -84,12 +84,18 @@ class SlackBridge:
             if w.startswith("@"):
                 zulip_msg["content"] = zulip_msg["content"].replace(w, "<" + w + ">")
 
+    def is_message_from_slack(self, msg: Dict[str, Any]) -> bool:
+        # Check whether or not this message is from Slack to prevent
+        # them from being tossed back to Zulip.
+        return msg["sender_email"] == self.zulip_config.get("integration_bot_email")
```

Comment:
If we're using the same Zulip bot user account for both, this can be checking if the sender is our own account, which would eliminate the need to add a new configuration variable.

Reply:
I updated this to fetch the generic bots email instead:
```python
self.zulip_config.get("email")
```